app/controllers/concerns/sign_in/authentication.rb

```diff
@@ -36,5 +39,16 @@ def authenticate_access_token
     def load_user
       SignIn::UserLoader.new(access_token: @access_token).perform
     end
+
+    def handle_authenticate_error(error)
+      context = { authorization: request.authorization }
+      log_message_to_sentry(error.message, :error, context)
+      sign_in_logger.authenticate_access_token_stats(:failure)
```

Comment:
Now that I think about it, be probably don't need an authenticate failure statsd either